openff/utilities/provenance.py

```diff
@@ -0,0 +1,33 @@
+import functools
+import re
+import subprocess
+from typing import Dict, Optional
+
+import pkg_resources
+
+
+@functools.lru_cache()
+def _get_conda_list_package_versions() -> Dict[str, str]:
+    """Returns the versions of any packages found while executing `conda list`."""
+
+    list_output = subprocess.check_output(["conda", "list"]).decode().split("\n")
```

Comment:
Any preference for the CLI vs API?

```suggestion
    from conda.cli.python_api import run_command

    list_output = run_command('list')[0].split("\n")
```

Reply:
Hmm at least for me conda wasn't importable on my local machine so I think sticking with CLI may be best for now